### DISCUSSION TITLE
"hidden" props for button doesn't actually hide?

### DISCUSSION THREAD

**Discussion by cktang88** [system-ui/theme-ui#1253] [Q&A] [answered]
I tried applying 
`<Button hidden={true}>hello</Button>`
but it seems the button is still visible? I was under the assumption it would be hidden with that attribute set...

Using v0.3.1

**Comment by atanasster** [comment, marked as answer]
@cktang88 - I believe this is a bug and you can enter it as a new issue - `<Button />` is forcing a `display: 'inline-block',` without checking for a `hidden` prop: https://github.com/system-ui/theme-ui/blob/f45a5a8fe59f2df4538fbe38438ba6da59467e99/packages/components/src/Button.js#L13

In the meantime you can work around it using css or sx props: 
`<Button css={{ visibility: "hidden" }}>Click me</Button>`
